scripts/gen_cfb_font_header.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Henrik Brix Andersen <henrik@brixandersen.dk>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import sys
+
+from PIL import ImageFont
+from PIL import Image
+from PIL import ImageDraw
+
+def generate_element(image, charcode):
+    """Generate CFB font element for a given character code from an image"""
+    blackwhite = image.convert("1", dither=Image.NONE)
+    pixels = blackwhite.load()
+
+    if args.dump:
+        blackwhite.save("{}_{}.png".format(args.name, charcode))
+
+    args.output.write("""\t/* {:d} */\n\t{{\n""".format(charcode))
```

Comment:
I suggest to print the character here.
```diff
-    args.output.write("""\t/* {:d} */\n\t{{\n""".format(charcode))
+    args.output.write("""\t/* {:c} */\n\t{{\n""".format(charcode))
```

Reply:
@jfischer-phytec-iot I did that in an initial revision, but it only works for ASCII printable character codes. The font generator does not restrict you to use printable characters.

Reply:
Agree, but it is very handy for the fonts, can we have both formats {:d} and {:c}? :smile: